spec/features/recipe_ingredients/new_recipe_ingredient_page_spec.rb

```diff
@@ -1,45 +0,0 @@
-feature 'user creates new recipe ingredient' do
```

Comment:
Why removed?

Reply:
Can't remember now but I got this tests covered in previous pull request